manager/src/main/java/com/redhat/service/bridge/manager/BridgesServiceImpl.java

```diff
@@ -117,4 +119,28 @@ public Bridge updateBridge(BridgeDTO bridgeDTO) {
         LOGGER.info("[manager] Bridge with id '{}' has been updated for customer '{}'", bridge.getId(), bridge.getCustomerId());
         return bridge;
     }
+
+    @Override
+    public BridgeDTO toDTO(Bridge bridge) {
```

Comment:
@kostola Same point here. These are inheriting the `@Transactional` from the class-level annotation. There is no need for a transaction on these methods, so could you please remove the class-level annotation and place `@Transactional` only on methods where it is required.

Reply:
Done :+1: